live-examples/css-examples/filter-effects/hue-rotate.html

```diff
@@ -0,0 +1,35 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="filter">
+<div class="example-choice">
+<pre><code id="example_one" class="language-css">filter: hue-rotate(0);</code></pre>
+<button type="button" class="copy hidden" aria-hidden="true">
+    <span class="visually-hidden">Copy to Clipboard</span>
+</button>
+</div>
+
+<div class="example-choice" initial-choice="true">
+<pre><code id="example_one" class="language-css">filter: hue-rotate(90deg);</code></pre>
```

Comment:
Again here, "example_one" -> "example_two" (and the subsequent examples need to increment as well).